### ISSUE TITLE
Getting error in publisher(_:didFailWithError:) -> 1541 - Timed out while attempting to publish

### ISSUE THREAD

**Issue by anis-applocum** [opentok/opentok-ios-sdk-samples#244]
My App requirement is to test Audio and Video quality first, then start video call. Network test is done properly.
And when I start actual call I'm getting above error. And unable to publish my feed and audio to subscriber.

publisher(_:didFailWithError:)
1541 - Timed out while attempting to publish.

https://github.com/opentok/opentok-network-test/tree/master/iOS-Sample

I've tried in opentok version 2.19.1, 2.20.0

**Comment by Adza93** [comment]
I am having the same issue, @anis-applocum  have you managed to work this out? Thanks for help :)

**Comment by anis-applocum** [comment]
No, the issue is still there. I changed the flow of my application.

**Comment by Adza93** [comment]
@anis-applocum what did you do to make it work? Completely removed test session part and just called main session or some other workaround. Thank you for your help!

**Comment by Adza93** [comment]
@anis-applocum what did you do to make it work? Completely removed test session part and just called main session or some other workaround. Thank you for your help!

**Comment by Adza93** [comment]
@anis-applocum could you please follow up with your answer? Thank you!

**Comment by anis-applocum** [comment]
I had to remove the network test in order to make my existing functionality work.

**Comment by Adza93** [comment]
> I had to remove the network test in order to make my existing functionality work.

OK that is not correct solution for me. You should perform network test if possible to ensure proper network connection before subscribing to session. I will post an update if I get any feedback from Vonage team.

**Comment by anis-applocum** [comment]
Thanks, let me know if you get the solution, and I'll implement it again.
